### DISCUSSION TITLE
ICC profiles

### DISCUSSION THREAD

**Discussion by f-saez** [blend2d/blend2d#209] [Q&A] [answered]
Hi,

I'm just curious. Is it possible to get the ICC profile of a JPEG opened with Blend2D ?
I mean, just the bytes embedded in the file, nothing more.

**Comment by kobalicek** [comment, marked as answer]
Not at the moment, but feel free to open an issue about this - I don't see a big deal supporting it.

**Comment by f-saez** [comment]
I don't need it right now, but knowing your open to do it is all I need to know.

thanks.
